firebase-firestore/src/main/java/com/google/firebase/firestore/local/RemoteDocumentCache.java

```diff
@@ -37,10 +35,10 @@
    * <p>The cache key is extracted from {@code maybeDocument.key}. If there is already a cache entry
    * for the key, it will be replaced.
    *
-   * @param maybeDocument A Document or NoDocument to put in the cache.
+   * @param document A Document or NoDocument to put in the cache.
```

Comment:
nit: The mention of `NoDocument` here is obsolete. Also, `Document` should not begin with an uppercase letter because it is no longer the name of a class. Also on lines 50 and 58.